### PR TITLE
[NAE-1682] Translate tooltip admin view

### PR DESCRIPTION
**Pull request by machacjozef**
# Description

- translate text

Implements [NAE-1682]

## Dependencies

No new dependencies were introduced

### Third party dependencies

No new dependencies were introduced

### Blocking Pull requests

There are no dependencies on other PR

## How Has Been This Tested?

This was tested manually.

### Test Configuration

| Name                | Tested on |
|---------------------| --------- |
| OS                  |      Linux Mint 20.3     |
| Runtime             |  Node v12.22.12         |
| Dependency Manager  |   NPM 6.14.16    |
| Framework version   |    Angular 13.3.1     |
| Run parameters      |           |
| Other configuration |           |

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes have been checked, personally or remotely, with @SamuelPalaj 
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have resolved all conflicts with the target branch of the PR
- [x] I have updated and synced my code with the target branch
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing tests pass locally with my changes:
    - [x] Lint test
    - [x] Unit tests
    - [x] Integration tests
- [x] I have checked my contribution with code analysis tools:
    - [x] [SonarCloud](https://sonarcloud.io/project/overview?id=netgrif_application-engine)
    - [x] [Snyk](https://app.snyk.io/org/netgrif)
- [x] I have made corresponding changes to the documentation:
    - [x] Developer documentation
    - [x] User Guides
    - [x] Migration Guides

[NAE-1682]: https://netgrif.atlassian.net/browse/NAE-1682?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by renczesstefan — DISMISSED** [review]

File: projects/netgrif-components/src/lib/admin/ldap-group-role-assignment/ldap-group-role-assignment.component.html [hunk 1]
```diff
@@ -22,10 +22,10 @@
                 </div>
 
                 <span style="flex: 1 1 auto;"></span>
-                <button mat-icon-button matTooltip="Select All" (click)="selectAllLdapGroup(true)">
+                <button mat-icon-button matTooltip="{{ 'side-menu.tooltip.SelectAll' | translate }}" (click)="selectAllLdapGroup(true)">
```
Comment (minop) [inline projects/netgrif-components/src/lib/admin/ldap-group-role-assignment/ldap-group-role-assignment.component.html:25]: why is the tooltip text saved under the `side-menu` key if it is not used in the side menu?

File: projects/netgrif-components/src/lib/data-fields/text-field/html-textarea-field/html-textarea-field.component.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <div class="height-100" #quillContainer>
     <quill-editor theme="snow" *ngIf="!formControlRef.disabled" [(ngModel)]="textAreaField.value"
-                  [modules]="quillModules" [bounds]="quillContainer"></quill-editor>
+                  [modules]="quillModules" [bounds]="quillContainer" placeholder="{{ 'dataField.textarea.insertText' | translate }}"></quill-editor>
```
Comment (minop) [inline projects/netgrif-components/src/lib/data-fields/text-field/html-textarea-field/html-textarea-field.component.html:3]: should this not use the placeholder attribute from the data field object?

**Review by minop — CHANGES_REQUESTED** [review]

File: projects/netgrif-components/src/lib/admin/ldap-group-role-assignment/ldap-group-role-assignment.component.html (hunk 1 above)
Comment (machacjozef) [reply projects/netgrif-components/src/lib/admin/ldap-group-role-assignment/ldap-group-role-assignment.component.html:25]: all those components are stored there in side-menu...    IDK

File: projects/netgrif-components/src/lib/data-fields/text-field/html-textarea-field/html-textarea-field.component.html (hunk 2 above)
Comment (machacjozef) [reply projects/netgrif-components/src/lib/data-fields/text-field/html-textarea-field/html-textarea-field.component.html]: Done

**Review by minop — APPROVED** [review]

File: projects/netgrif-components/src/lib/admin/ldap-group-role-assignment/ldap-group-role-assignment.component.html (hunk 1 above)
Comment (minop) [reply projects/netgrif-components/src/lib/admin/ldap-group-role-assignment/ldap-group-role-assignment.component.html:25]: my bad I mixed up the names of the panel on the left and ride side of the screen....

**Review by Kovy95 — APPROVED** [review]

**Review by mladoniczky — APPROVED** [review]

**Pull request merged**